Modelica/StateGraph.mo

```diff
@@ -1535,9 +1535,9 @@ buttons:
                        annotation (Placement(transformation(extent={{55,-50},{
                   65,-40}})));
       Real level(start=0,fixed=true) "Tank level in % of max height";
-      parameter Real A=1 "Ground area of tank in m^2";
-      parameter Real a=0.2 "Area of drain hole in m^2";
-      parameter Real hmax=1 "Max height of tank in m";
+      parameter Real A(unit="m2")=1 "Ground area of tank";
+      parameter Real a(unit="m2")=0.2 "Area of drain hole";
+      parameter Real hmax(unit="m")=1 "Max height of tank";
```

Comment:
```suggestion
      parameter Modelica.Units.SI.Area A=1 "Ground area of tank";
      parameter Modelica.Units.SI.Area a=0.2 "Area of drain hole";
      parameter Modelica.Units.SI.Height hmax=1 "Max height of tank";
```

Reply:
Accepting. At first I thought that they were deliberately not using that SI-package, but looking at the other models I realize I was wrong.

Reply:
Can use `SI.Area` and `SI.Height` for short.

Note, that `SI.Height` is bounded by zero. `SI.Length` would be the unbounded case.